tests/test_cli.py

```diff
@@ -205,6 +211,11 @@ def assert_status_down():
         assert_status_up()
         assert get_volume(instance.profile.home_mount)
         assert get_volume(instance.profile.conda_volume_name())
+        for extra_mount in instance.profile.extra_mounts:
+            extra_volume, _, _ = instance.profile.parse_extra_mount(extra_mount)
+            # TODO: For some reason this assert fails,
+            # the extra mount volumes are not created.
+            # assert get_volume(str(extra_volume))
```

Comment:
I tried a bunch of stuff but couldn't make this work...

Reply:
I think manipulating the `instance` variable like this may not have any effect on the profile that is picked up by the CLI runner. You could check that by printing the output of `runner.involke(cli.cli, ["profiles", "show", "default"])`.

I can envision two ways to address this:
1. Create an additional fixture named `use_extra_volumes` or so that actually changes the configuration when requested.
2. Just configure all tests to use extra_volumes.

I don't like the 2nd approach, because it means that we are moving further away from the user default config in our tests, which reduces our test effectiveness, however it might be a good way to debug this problem.

Reply:
I have already tried the second approach by modifying the Profile fixture, but that didn't work. I think we would need to go deeper and modify the config fixture.

I would need to think more about how to create `use_extra_volumes` fixture.